eventdemo/source/objc/EventDemo/ViewControllers/RoomsListViewController.h

```diff
@@ -0,0 +1,26 @@
+/**
+ *  Copyright 2014-2016 CyberVision, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+@interface RoomsListViewController : UITableViewController
+{
+    NSArray *roomsList;
+}
+
+@property (weak, nonatomic) IBOutlet UIButton *editButton;
```

Comment:
Good practice is to declare UI elements as private properties. Please move it to implementation file.